internal/pkg/cli/env_init.go

```diff
@@ -470,8 +470,8 @@ https://aws.amazon.com/premiumsupport/knowledge-center/ecs-pull-container-api-er
 		publicSubnets, err := o.selVPC.PublicSubnets(envInitPublicSubnetsSelectPrompt, "", o.importVPC.ID)
 		if err != nil {
 			if err == selector.ErrSubnetsNotFound {
-				log.Warningf(`No existing public subnets were found in VPC %s.
-If you proceed without public subnets, you will not be able to deploy Load Balanced Web Services in this environment.
+				log.Warningf(`No existing subnets were found in VPC %s.
+If you proceed without specifying public subnets, you will not be able to deploy Load Balanced Web Services in this environment.
```

Comment:
Should we say "specifying two public subnets"?

Reply:
yeah maybe "without at least two public subnets"